CONTRIBUTING.MD

```diff
@@ -59,6 +62,12 @@ The format can be described more formally as follows:
 ```
 
 The first line is the subject and should be no longer than 70 characters, the second line is always blank, and other lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
+### Commit Rules: 
```

Comment:
I don't think this new section is needed, we should just update the format of the commit section, and keep this info/links the same as the OLM project. See [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/CONTRIBUTING.md#commit-messages-and-sign-off-dco) for what to update.